api/src/API/Controllers/RaceResultsController.cs

```diff
@@ -38,12 +39,28 @@ public async Task<IActionResult> GetAllRaceResults(string orgId, string memberId
         [HttpGet("/organizations/{orgId}/raceresults")]
         public async Task<IActionResult> GetAllRaceResults(string orgId)
         {
+            RaceContainerClient raceContainer = containerProvider.RaceContainer;
             MemberContainerClient memberContainer = containerProvider.MemberContainer;
+
             IEnumerable<Member> members = await memberContainer.GetAllMembersAsync(orgId);
             IEnumerable<string> memberIds = members.Select(member => member.Id.ToString());
 
             RaceResultContainerClient container = containerProvider.RaceResultContainer;
-            IEnumerable<RaceResult> result = await container.GetRaceResultsForMembersAsync(memberIds);
+            IEnumerable<RaceResult> raceResults = await container.GetRaceResultsForMembersAsync(memberIds);
+
+            var racesNeeded = raceResults.Select(result => result.RaceId).ToHashSet();
+            var membersNeeded = raceResults.Select(result => result.MemberId);
+
+            var membersInResponse = (await memberContainer.GetMembersAsync(orgId, membersNeeded));
```

Comment:
I think we should stick with a unified method naming schema. If we're naming methods that return dictionaries "AsDict", we should continue using that convention.

Reply:
A member might not have a race result, so we don't need to fetch their record